plan/newplanbuilder.go

```diff
@@ -522,18 +636,45 @@ func (b *planBuilder) buildNewSelect(sel *ast.SelectStmt) Plan {
 		}
```

Comment:
From will never be nil after implemented table dual, so else block can be removed.